src/GovUk.Education.ExploreEducationStatistics.Publisher/PublisherHostBuilderExtensions.cs

```diff
@@ -123,6 +123,7 @@ public static IHostBuilder ConfigurePublisherHostBuilder(this IHostBuilder hostB
                     .AddScoped<IIndicatorRepository, IndicatorRepository>()
                     .AddScoped<IPublishingCompletionService, PublishingCompletionService>()
                     .AddScoped<IPublicationRepository, PublicationRepository>()
+                    .AddScoped<IReleaseRepository, ReleaseRepository>()
```

Comment:
I didn't realise `IReleaseRepository` was needed at first, but it's because it's used indirectly by `Content.Services.PublicationCacheService` -> `Content.Services.IPublicationService` -> `IReleaseRepository` which is why I added this in.

I also checked `IReleaseVersionRepository` is still needed, and that one is used by `Publisher.Services.ReleaseService` which uses method `GetPublishedDate`.

Reply:
ah ok, makes sense :)